servant-auth-server/src/Servant/Auth/Server/Internal/Cookie.hs

```diff
@@ -142,18 +144,27 @@ acceptLogin cookieSettings jwtSettings session = do
       xsrfCookie <- makeXsrfCookie cookieSettings
       return $ Just $ addHeader sessionCookie . addHeader xsrfCookie
 
--- | Adds headers to a response that clears all session cookies.
+expireTime :: UTCTime
```

Comment:
A little comment eplaining what this is, why this value, what it's used for would be great if you don't mind. Even though we can somewhat read that logic below, but IMO it's not "direct enough" to prevent the need for a comment.

Reply:
Added, let me know if it suffices.

Reply:
Great, thanks.